packages/react-native-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/DeferredEventEmitter.java

```diff
@@ -121,4 +122,8 @@ public void onMediaFileUploadSucceeded(int mediaId, String mediaUrl, int mediaSe
     public void onMediaFileUploadFailed(int mediaId) {
         setMediaFileUploadDataInJS(MEDIA_UPLOAD_STATE_FAILED, mediaId, null, 0);
     }
+
+    public void updateCapabilities(WritableMap capabilities) {
```

Comment:
This is definitely a nitpick, but just wondering if you considered maybe passing the `GutenbergProps` object here instead of just the `WritableMap`. I feel like that might be slightly better just because it keeps us using the typesafe `GutenbergProps` object (instead of the `WritableMap` which has no type safety) as long as possible. Definitely feel free to leave this as-is if you prefer.

Reply:
Thanks for the tip. Fixed.

Reply:
@mchowning This one is also fixed and ready for the second look.

Reply:
Looks good! 👍 Thanks @marecar3 !